weareinreach/inreach-catalog
# Feature/1200543167522215/org social media icons

## Description

Adding social media icons to catalog according to Figma mockup. See Asana ticket for link to mockup.

## Asana ticket: https://app.asana.com/0/1132189118126148/1200543167522215

## PR Checklist




## How to Test

1) In Control Panel, navigate to an organization's page, scroll down the "Social Media" section, and make sure there is at least one social media profile present. If none are present, add one.

2) Visit organization's page in catalog. In desktop view, check that social media icons appear as in the mockup.

3) Verify mobile view by resizing window, visiting from mobile device, or using responsive mode in browser.

4) Try this with different combinations of social media profiles.
